templates/appliance/plex/index.html

```diff
@@ -162,4 +170,6 @@ <h2>Build your own appliance</h2>
   </div>
 </section>
 
-{% endblock content %}
+<script src="{{ versioned_static('js/build/appliance.min.js') }}" defer></script>
```

Comment:
Can you add this via the appliance base template so this script is used across the pages.